src/components/objectListing/objectListing.component.tsx

```diff
@@ -331,13 +335,33 @@ export const ObjectList = ({
   });
   const { virtualItems: virtualRows, totalSize } = rowVirtualizer;
 
+  const primaryKey = draggedObject && getPrimaryKey(draggedObject);
+
   return (
     <div
       className={clsx(
         "flex h-full flex-col space-y-4",
         isPanelOpen ? "lg:space-y-8" : "md:space-y-8",
       )}
     >
+      <DragOverlay modifiers={[snapCenterToCursor]}>
+        {draggedObject ? (
+          <div className="my-o flex max-w-[350px] items-center space-x-2 ">
```

Comment:
Move to a component, add a white background, `p` text should b one size smaller. Rectangle, maybe a very light grey 1px border, no border-radius.